src/Engine/EngineBase.php

```diff
@@ -29,7 +29,7 @@ abstract class EngineBase
         'de-frk' => 'Deutsch (Fraktur)',
         'enm' => 'Middle English (1100-1500)',
         'es-old' => 'español (viejo)',
-        'equ' => 'Math / equation detection module',
+        'frm' => 'français moyen (1400-1600)',
```

Comment:
I think this should be "moyen français", according to https://fr.wikipedia.org/wiki/Moyen_fran%C3%A7ais

Reply:
I mean, "moyen français (1400-1600)", the swap is the important thing.

Reply:
Done